use-cases/client-server-msgs-perf/svmsg_file_server.c

```diff
@@ -295,10 +302,18 @@ main(int argc, char *argv[])
             elapsed_ns = (nsec1 - nsec0);           // Elapsed-ns for this op
 
 #if L3_ENABLED
+
             // Record time-consumed. (NOTE: See clarification below.)
+#  if L3_FASTLOG_ENABLED
+            l3_log_fast("Server msg: Increment: ClientID=%d, "
+                        "Thread-CPU-time=%" PRIu64 " ns. (L3-fast-log)",
```

Comment:
@gregthelaw - This is a mystery.

All the right env-vars and `-D flags` are being passed thru the build system.

I changed the message to say `(L3-fast-log)`, intentionally.

And I manually verified in CI and my dev-runs that this term **is coming** out, from L3-dump utility. 

So **L308 is being invoked**, but how come the overall throughput metric in some experiments is  slower for L3-fast v/s L3-slow logging?

That's very strange. I would have expected that with this change the overall throughput metric for this new case will be as good as, or better than, the plain L3-logging.

All this perf measurements on my Linux-VM is becoming unreliable. We really need to run these tests on your Linux laptop directly.

Reply:
Yes, I too would have expected l3-fast to be faster! Maybe the compiler generates code that is more VM-friendly. Who knows? Will have to try it on my native Linux laptop to see.